Sources/Client/ScoreboardView.cpp

```diff
@@ -241,6 +257,50 @@ namespace spades {
 			int row = 0, col = 0;
 			float colWidth = (float)width / (float)cols;
 			
+			//definite a palette of 32 color in RGB code
+			int palette[32][3]={
+						{0,0,0},//0 black
+						{255,255,255},//1 white
+						{128,128,128},//2 grey
+						{255,255,0},//3 yellow
+						{0,255,255},//4 cyan-acqua
+						{255,0,255},//5 fuchsia						
+						{255,0,0},//6 red
+						{0,255,0},//7 lime
+						{0,0,255},//8 blue
+						
+						{128,0,0},//9 maroon
+						{0,128,0},//10 green
+						{0,0,128},//11 navy
+						{128,128,0},//12 olive
+						{128,0,128},//13 purple
+						{0,128,128},//14 teal
+						
+						{255,128,0},//15 orange
+						{255,0,128},//16 deep pink
+						{128,0,255},//17 violet
+						{0,128,255},//18 bluette
+						{128,255,0},//19 lime 2
+						{0,255,128},//20 spring green
+						
+						{255,128,128},//21 light pink
+						{128,255,128},//22 light spring
+						{128,128,255},//23 light blue			
+						{128,255,255},//24 light azure
+						{255,255,128},//25 light yellow
+						{255,128,255},//26 light pink2
+						
+						{165,42,42},//27 brown
+						{255,69,0},//28 orange	red
+						{255,165,0},//29  orange
+						
+						{139,69,19},//30 maroon medium
+						{210,105,30},//31 choccolate
```

Comment:
yes i know, but i dont know  whether it is better position to put it in: MapView.cpp or ScoreboardView.cpp.
Second question. This requires something in particular about the place, or the syntax, where it needs to be declared the color palette?